server/belga/search_providers.py

```diff
@@ -369,14 +373,34 @@ def format_list_item(self, data):
             'creditline': get_text(data['credit']),
             'source': get_text(data['source']),
             'language': get_text(data['language']),
-            'abstract': get_text(self._get_abstract(data)),
-            'body_html': self._get_body_html(data),
+            'body_html': self._get_abstract(data) + self._get_body_html(data),
             'extra': {
                 'bcoverage': guid,
             },
             '_fetchable': False,
+            'keywords': data.get("keywords"),
+            'sign_off': self.get_sign_off(data.get('authors')),
+            'subject': self.get_subjects(data.get('keywords'))
         }
 
+    def get_sign_off(self, authors):
+        if not authors:
+            return
+        return ', '.join(map(lambda x: x['name'] + '/' + x['type'], authors))
```

Comment:
imo it should be only names joined by `/`

Reply:
According to a comment mentioned by Philippe on the ticket, the sign-off should be like `PHD/Admin`, here **Admin** is the status of the author.

Reply:
admin is author role?

Reply:
Yep, on the preview of sign_off, it will show like `name/type` so here it is like `authors : [{'name': 'PHD', 'type': 'ADMIN'}]`

Reply:
it's usually sign off of multiple users, so more like `author1/author2/author3` than `author1/role1`

Reply:
supppse we have mulitple author for example `authors : [{'name': 'PHD', 'type': 'ADMIN'}, {'name': 'BRV', 'type': 'AUTHOR'}]`  so its sign off  will be like `PHD/Admin, BRV/Author`

Reply:
yep but if you would do the same thing via superdesk it would be `PHD/BRV` so not sure what is the expectation

Reply:
I see, but then when we use such story with different sign off format how does it work in the output?

Reply:
With superdesk we have sign-off `PHD/BRV`
With Belga archive will have sign off `PHD/Admin, BRV/Author` (as per the request of Phillippe here we always get the response in list format `[{'name': 'PHD', 'type': 'ADMIN'}, {'name': 'BRV', 'type': 'AUTHOR'}]` )

Reply:
> With superdesk we have sign-off `PHD/BRV` With Belga archive will have sign off `PHD/Admin, BRV/Author` (as per the request of Phillippe here we always get the response in list format `[{'name': 'PHD', 'type': 'ADMIN'}, {'name': 'BRV', 'type': 'AUTHOR'}]` )
do you know any other format for Belga archive sign-off?

Reply:
I'm talking about output format, if we get such article from belga archive and publish it what will be there? does it match the output of article created via superdesk?